site/docs/components/date-range-picker.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Date Range Picker
+navTitle: Date Range Picker
+headerImage: date-picker
+demoStoryId: date-picker
+githubLabels: ["component:date-picker"]
```

Comment:
This will live inside of the `date-picker/` package so I don't think it warrants its own label but happy to be talked out of that.